### PR TITLE
Database connection and model schema

### PR DESCRIPTION
**Pull request by ManishKumar0007**
create database connection and create schema for registration

### CODE REVIEW

File: backend/src/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import express from 'express';
+import cors from 'cors';
+import helmet from 'helmet';
```
Comment (aman-quo) [inline backend/src/index.js:6]: Read about the helmet.

File: backend/src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const port = process.env.APP_PORT;
+const api_version = process.env.API_VERSION;
+
+app.use(cors());
+app.use(helmet());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(morgan('combined', { stream: logStream }));
+
+database();
```
Comment (aman-quo) [inline backend/src/index.js:30]: @ManishKumar0007 
start the DB connection after the server starts listening on the configured port.

Comment (mkaubr007) [reply backend/src/index.js:30]: done

File: backend/src/index.js (hunk 1 above)
Comment (mkaubr007) [reply backend/src/index.js:6]: done

**Pull request merged**